.github/workflows/otbr.yml

```diff
@@ -73,7 +73,8 @@ jobs:
     - name: Bootstrap
       run: |
         sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
-        sudo apt-get --no-install-recommends install -y python3-setuptools python3-wheel ninja-build socat lcov
+        sudo apt-get --no-install-recommends install -y python3-setuptools python3-wheel ninja-build lcov
+        sudo bash script/install_socat
```

Comment:
Is it possible to install the desired version using something like "apt-get install package=version"?

Reply:
Specifying version is possible. I also tried to do so at first. However I found that there is usually only one available version (which is the latest one).